api/src/pcapi/core/offerers/models.py

```diff
@@ -740,9 +740,10 @@ class GooglePlacesInfo(PcObject, Base, Model):
         BigInteger, ForeignKey("venue.id", ondelete="CASCADE"), nullable=False, index=True, unique=True
     )
     venue: sa_orm.Mapped[Venue] = relationship("Venue", foreign_keys=[venueId], back_populates="googlePlacesInfo")
-    placeId = Column(Text, nullable=False, unique=True)
+    placeId: str | None = Column(Text, nullable=True, unique=True)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Je crois que dans ce cas mypy infère correctement le type
(On peut laisser le typehint explicite, c'est plus pour info)